src/webviews/webview-side/ipywidgets/kernel/index.ts

```diff
@@ -125,6 +132,9 @@ function renderIPyWidget(
     }
     const output = document.createElement('div');
     output.className = 'cell-output cell-output';
+    if (typeof model._vsc_test_cellIndex === 'number') {
```

Comment:
Since this attribute will only appear in tests, having this code in prodcution should be fine.
Note: This will be removed once we implement #9629